tests/testthat/test-renderer4-geoms-zoom.R

```diff
@@ -0,0 +1,97 @@
+acontext("zooming of geoms")
+
+# Plots with axis updates
+mtcars$cyl <- as.factor(mtcars$cyl)
+
+no_updates <- ggplot()+geom_point(aes(mpg, disp, 
+                                      colour=cyl), 
+                                  data = mtcars)
+update_x <- no_updates+
+  theme_animint(update_axes=c("x"))
+update_y <- no_updates+
+  theme_animint(update_axes=c("y"))
+update_xy <- no_updates+
+  theme_animint(update_axes=c("x","y"))
+
+viz <- (list(g=no_updates, 
+             g1=update_x, 
```

Comment:
instead of g, g1, g2, can you please change to more informative names? e.g. `neither` `x` `y` `both`